assets/src/components/activities/adaptive/components/common/PartComponent.tsx

```diff
@@ -24,7 +24,7 @@ const PartComponent: React.FC<AuthorProps | DeliveryProps> = (props) => {
 
   let height = props.model.height;
   // TODO: figure out how to default TF *only* to height auto without hard coding type
-  if (props.type === 'janus-text-flow' || props.type === 'janus-mcq') {
+  if (props.type === 'janus-mcq') {
```

Comment:
yeah.. so when I removed this I got the issue in feedback textflows, but what i found is setting the height to "auto" and not applying height property was acting same so I am not applying height to janus-text-flow elements. Height : "auto" was creating issues in case of animations like lucia in phases of the moon